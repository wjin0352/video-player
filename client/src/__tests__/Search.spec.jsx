import React from 'react';
import { shallow } from 'enzyme';
import SearchBar from '../components/search_bar';
import MainPage from '../components/main_page';
import VideoList from '../components/video_list';

// test is global jest function
test('Searchbar renders correctly', () => {
  const component = shallow(<SearchBar/>)
  expect(component).toMatchSnapshot();
});



