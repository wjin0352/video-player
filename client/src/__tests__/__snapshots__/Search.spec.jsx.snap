// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbar renders correctly 1`] = `
<div
  className="search-bar"
>
  <div
    className=""
    id="title_container"
  >
    <h2
      id="title"
    >
      Video Saver
    </h2>
    <p>
      Search for videos faster without ads then save them to a list for easy viewing later!
    </p>
  </div>
  <div
    className="divider"
  />
  <div
    className="input_container"
  >
    <input
      id="search_input"
      onChange={[Function]}
      placeholder=" Type in your search here "
      value=""
    />
  </div>
</div>
`;
